app/views/events/index.html.erb

```diff
@@ -1,26 +1,57 @@
 <%= link_to 'New Event', new_event_path %>
-<h1>All Events</h1>
-<table border="1">
-  <tr>
-    <th>Logo</th>
-    <th>Title</th>
-    <th>Date</th>
-    <th>Time Start</th>
-    <th>Time End</th>
-  </tr>
-  <% @events.each do |event| %>
-    <tr>
-      <td><%= event.logo %></td>
-      <td><%= event.title %></td>
-      <td><%= event.date %></td>
-      <td><%= event.time_start %></td>
-      <td><%= event.time_end %></td>
-
-      <td><%= link_to 'Show', event_path(event) %></td>
-      <td><%= link_to 'Edit', edit_event_path(event) %></td>
-      <td><%= link_to 'Destroy', event_path(event),
+
+<div class="event-view">
+  <div class="container mt-3">
+    <div class="d-flex justify-content-between col-min-width p-0">
+      <span class="heading">Events</span>
+      <%= button_to 'Create Event', new_event_path, method: :get, class: "button" %>
+      <!--<button class="button">Create Event</button>-->
+    </div>
+    <div class="row d-flex justify-content-between">
+
+      <% @events.each do |event| %>
+
+        <div class="col-md col-sm-12 col-xs col-min-width mx-3 box p-3">
```

Comment:
Use bootstrap classes for layout to auto adjust with respect to different screen sizes: 
`col-12 col-sm-6 col-md-4 col-lg-3`